pytential/symbolic/execution.py

```diff
@@ -932,6 +934,37 @@ def __repr__(self):
     def __str__(self):
         return f"{type(self).__name__}({repr(self.places)})"
 
+
+_KNOWN_GEOMETRY_COLLECTION_CACHE_KEYS = (
+        EvaluationMapperCSECacheKey,
+        _GeometryCollectionConnectionCacheKey,
+        _GeometryCollectionDiscretizationCacheKey,
+        )
+
+
+def add_geometry_collection(
```

Comment:
```suggestion
def add_geometry_to_collection(
```

Reply:
Renamed and added docs in https://github.com/inducer/pytential/compare/9d5eddbb52bc99495736f6349a9531e1463af545..1a41e0ba2726a623d153084ade2fbd20688cf07a.

Reply:
Reworded that a bit in https://github.com/inducer/pytential/compare/547c7d34855dff898f6fba4f4ea4899a0ae3d708..d623df0e16902c87f21c7bc05a7967db5a3e0836 (probably best to just check d623df0)